webapp/channels/src/components/apps_form/apps_form_container.tsx

```diff
@@ -31,6 +31,7 @@ type Props = {
 
 type State = {
     form?: AppForm;
+    updateType?: 'submit' | 'refresh';
```

Comment:
Should we turn `submit` and `refresh` into enums? I see them being used in a number of places

Reply:
Agree...done.